config/webpack/shared.js

```diff
@@ -47,14 +47,15 @@ module.exports = {
     new webpack.optimize.CommonsChunkPlugin({
       name: 'common',
       minChunks: (module, count) => {
-        if (module.resource && /node_modules\/react-intl/.test(module.resource)) {
+        const reactIntlPathRegexp = new RegExp(`node_modules\\${sep}react-intl`);
```

Comment:
I thought too but you need an actual escaping in the regexp string to avoid confusions with a regexp class(\d \w...).